media/css/mozorg/home/home-mr1-promo.scss

```diff
@@ -82,6 +82,16 @@ $image-path: '/media/protocol/img';
     margin-top: $spacing-sm;
     position: relative;
     z-index: 11;
+
+    a:link,
+    a:visited {
+        color: get-theme('body-text-color');
+
+        &:hover,
+        &:active {
+            color: get-theme('link-color-hover');
```

Comment:
From a consistency point of view, this is the only link on the page that seems to change color on hover/focus. Maybe that's ok? I'll leave the call to you

Reply:
Oh, you're right. Maybe the underline is enough. I'll fix.